action.yml

```diff
@@ -8,6 +8,7 @@ inputs:
     required: true
   architecture:
```

Comment:
I don't if we really need this at all, if we're just checking it in code.

Reply:
The yaml syntax checker complains when it's not there.